app/jobs/bulk_action_jobs.rb

```diff
@@ -272,4 +272,14 @@ def perform(collection_id, overwrite = true, save_field = nil)
       [successes, errors]
     end
   end
+
+  class ReturnCheckouts < ActiveJob::Base
+    queue_as :bulk_return_checkouts
+    def perform(collection_id, _params)
+      collection = Admin::Collection.find(collection_id)
+      collection.media_object_ids.each do |id|
+        Checkout.active_for_media_object(id)&.update(return_time: DateTime.current)
+      end
```

Comment:
I was thinking about how to make this more efficient and found a couple things:

`active_for_media_object` can take an array which will do one query like `WHERE checkout.media_object_id IN ['id1', 'id2', 'id3']`.  I'm not sure if we'll run into issues with collections with tons of media objects though.

`update_all` will do a single SQL update using the WHERE clauses setup in `active_for_media_object` instead of one update (and transaction) per checkout.  Things to note are that  `update_all` doesn't run validations and callbacks (which seems fine in this case) and it doesn't update the `updated_at` timestamp.  We can manually update `updated_at` as part of the `update_all` though and it will be close enough to the timestamp Rails would have given it.

```suggestion
      Checkout.active_for_media_object(collection.media_object_ids).update_all(return_time: DateTime.current, updated_at: DateTime.current)
```